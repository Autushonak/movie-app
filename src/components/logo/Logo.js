import React from 'react';
import styled from 'styled-components';
import { colors } from '../../constants/colors';

const LogoStyled = styled.div`
  display: flex;
  flex-direction: row;
`;
const LogoPart1 = styled.h3`
  margin: 0;
  color: ${colors.PINK};
`;
const LogoPart2 = styled.h3`
  margin: 0;
  color: ${colors.PINK};
  font-weight: 300;
`;

const Logo = () => (
  <LogoStyled>
    <LogoPart1>netflix</LogoPart1>
    <LogoPart2>roulette</LogoPart2>
  </LogoStyled>
);

export default Logo;
