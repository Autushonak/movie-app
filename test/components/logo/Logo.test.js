import 'jest-styled-components';
import React from 'react';
import { mount } from 'enzyme';
import Logo from '../../../src/components/logo/Logo';

describe('Logo ', () => {
  test('should be rendered correctly', () => {
    const component = mount(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
