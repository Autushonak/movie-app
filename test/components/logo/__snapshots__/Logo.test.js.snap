// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo  should be rendered correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  margin: 0;
  color: #F65261;
}

.c2 {
  margin: 0;
  color: #F65261;
  font-weight: 300;
}

<Logo>
  <styled.div>
    <div
      className="c0"
    >
      <styled.h3>
        <h3
          className="c1"
        >
          netflix
        </h3>
      </styled.h3>
      <styled.h3>
        <h3
          className="c2"
        >
          roulette
        </h3>
      </styled.h3>
    </div>
  </styled.div>
</Logo>
`;
